packages/email/template-components/template-footer.tsx

```
@@ -20,11 +23,24 @@ export const TemplateFooter = ({ isDocument = true }: TemplateFooterProps) => {
-        Documenso, Inc.
-        <br />
-        2261 Market Street, #5211, San Francisco, CA 94114, USA
-      </Text>
+      {branding.brandingCompanyDetails ? (
+        <Text className="my-8 text-sm text-slate-400">
+          {branding.brandingCompanyDetails.split('\n').map((line, idx) => {
+            return (
+              <>
+                {idx > 0 && <br />}
+                {line}
+              </>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)